src/test/ui/consts/const-typeid-of.rs

```diff
@@ -1,8 +0,0 @@
-use std::any::TypeId;
-
-struct A;
-
-fn main() {
-    const A_ID: TypeId = TypeId::of::<A>();
-    //~^ ERROR `std::any::TypeId::of` is not yet stable as a const fn
-}
```

Comment:
Please make this "run-pass" and check that it works.

Reply:
(Unless we already have another test that does that -- I am not sure if we do.)

Reply:
Ah we do have this one already: https://github.com/rust-lang/rust/blob/master/src/test/ui/consts/const-typeid-of-rpass.rs